### ISSUE TITLE
Backwards Compatible Calls cause exception

### ISSUE THREAD

**Issue by nduitz** [danschultzer/pow#451]
Using [pow_postgres_store](https://github.com/ZennerIoT/pow_postgres_store) the first time logging in after application startup causes an exception:

`function Pow.Postgres.Store.keys/1 is undefined or private`

The issue here seems to be:
`defp has_binary_keys?(store) do: not function_exported?(store, :all, 2)`

Documentation states: 
```
Note that this function does not load the module in case it is not loaded.
Check Code.ensure_loaded/1 for more information.
```

So simply calling `Code.ensure_loaded(store)` upfront fixes the issue.

To reproduce this behaviour setup [pow_postgres_store](https://github.com/ZennerIoT/pow_postgres_store) as store and try a login directly after application startup.

**Comment by nduitz** [comment]
See MR: https://github.com/danschultzer/pow/pull/452

**Comment by danschultzer** [comment]
Thanks! 🙏 

The PR looks good though I try to get around all `Code.ensure_compiled?/1` and `Code.ensure_loaded?/1` calls due to https://github.com/danschultzer/pow/issues/329. Guess Pow 1.1.0 can't come soon enough now! I want to review a few alternative ideas but otherwise get this merged.

Out of curiosity, does this only occur if you start up with `mix phx.server` instead of running a release?

**Comment by danschultzer** [comment]
Solved with #452 8e5a20fbd7decc1243109f0ed40fe360043f35c6

**Comment by nduitz** [comment]
> Out of curiosity, does this only occur if you start up with `mix phx.server` instead of running a release?

I can confirm that this does only occur running mix phx.server.

But thanks for merging this <3
